### PR TITLE
feat(nav popover): added the headline and closing button for the popover

### PR DESCRIPTION
**Pull request by madaluca**

### CODE REVIEW

File: packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx [hunk 1]
```diff
@@ -40,9 +41,25 @@ export class NavigationPopover implements ComponentInterface {
...
 
+  /**
+   * Defines the heading of the popover
+   */
+  @Prop() heading?: string
+
+  /**
+   * If 'false', the closing button is not rendered
+   */
+  @Prop() hasClosingBtn = true
```
Comment (hirsch88) [inline packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx]: - [x] not short forms. please write closable

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx (hunk 1 above)
Comment (madaluca) [reply packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx]: done

File: packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx [hunk 2]
```diff
@@ -40,9 +41,25 @@ export class NavigationPopover implements ComponentInterface {
...
 
+  /**
+   * Defines the heading of the popover
+   */
+  @Prop() heading?: string
+
+  /**
+   * If 'false', the closing button is not rendered
+   */
+  @Prop() isClosable = true
```
Comment (hirsch88) [inline packages/components/src/components/bal-navigation/bal-navigation-popover/bal-navigation-popover.tsx]: - [x] just closable no is-
- [x] and build again the components.d.ts is still old

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts [hunk 3]
```diff
@@ -448,5 +448,29 @@ NavigationPopover.args = {
   activeColor: 'primary',
   inactiveColor: 'light',
   inverted: false,
+  hasClosingBtn: false,
```
Comment (hirsch88) [inline packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts]: - [x] change to closable

**Review by hirsch88 — CHANGES_REQUESTED** [review]

**Review by hirsch88 — APPROVED** [review]

**Pull request merged**